ntia_conformance_checker/sbom_checker.py

```diff
@@ -89,16 +89,10 @@ def get_components_without_suppliers(self, return_tuples=False):
         """Retrieve name and/or SPDX ID of components without suppliers."""
         components_without_suppliers = []
         for package in self.doc.packages:
-            # both package supplier and package originator satisfy the "supplier"
-            # requirement
-            # https://spdx.github.io/spdx-spec/v2.3/package-information/#76-package-originator-field
-            no_package_supplier = package.supplier is None or isinstance(
+            no_supplier = package.supplier is None or isinstance(
```

Comment:
I simplified the variable naming here, changing `no_package_supplier` to `no_supplier`